docs/guides/setup/configuring_metadata_stores/how_to_configure_a_metricsstore.md

```diff
@@ -1,75 +1,110 @@
+  In theory, any valid StoreBackend can be used, however at the time of writing, the only BackendStore under test for use with a MetricStore is the DatabaseStoreBackend with Postgres.
+  To use an SQL Database like Postgres, provide two fields: `class_name`, with the value of `DatabaseStoreBackend`, and `credentials`.
+  Credentials can point to credentials defined in your `config_variables.yml`, or alternatively can be defined inline.
 
 ```yaml
 stores:
     #  ...
-    metrics_store:  # You can choose any name for your metric store
+    metric_store:  # You can choose any name as the key for your metric store
         class_name: MetricStore
         store_backend:
             class_name: DatabaseStoreBackend
-            # These credentials can be the same as those used in a Datasource configuration
             credentials: ${my_store_credentials}
+            # alternatively, define credentials inline:
+            # credentials:
+            #  username: my_username
+            #  password: my_password
+            #  port: 1234
+            #  host: xxxx
+            #  database: my_database
+            #  driver: postgresql
 ```
 
-
 The next time your DataContext is loaded, it will connect to the database and initialize a table to store metrics if
 one has not already been created. See the metrics_reference for more information on additional configuration
 options.
 
 ### Configuring a Validation Action
 
-Once a MetricStore is available, it is possible to configure a new `StoreMetricsAction` to save metrics during
-validation.
-
-Add the following yaml block to your DataContext validation operators configuration:
+Once a MetricStore is available, a `StoreMetricsAction` Validation Action can be added to your Checkpoint in order to save metrics during
+validation. This Validation Action has three required fields:
+- The `class_name` field determines which class will be instantiated to execute this action, and must be `StoreMetricsAction`.
+- The `target_store_name` field defines which Store backend to use when persisting the metrics. This should match the key of the MetricStore you added in your `great_expectations.yml`, which in our example above is `metrics_store`.
+- The `requested_metrics` field identifies which Expectation Suites and metrics to store. Please note that this API is likely to change in a future release.
+  Validation Result statistics are available using the following format:
+    ```yaml
+      expectation_suite_name:
+        statistics.<statistic name>
+   ```
+  Values from inside a particular Expectation's `result` field are available using the following format:
+   ```yaml
+      expectation_suite_name:
+        - column:
+          <column name>:
+            <expectation name>.result.<value name>
+   ```
+  In place of the Expectation Suite name, you may use `"*"` to denote that any expectation suite should match. 
```

Comment:
Could you explain this a bit more? Is it that if the Action specifies and Expectation Suite name, it will only write Metrics if run with that suite, but if * is used, then it will write Metrics if run with any suite?

Reply:
correct - I'll update to make it clearer